static/docs/get-started/index.md

```diff
@@ -1,26 +1,20 @@
 # Get Started
 
-_Get Started_ is a step-by-step introduction into basic DVC concepts. It doesn't
+_Get Started_ is a step-by-step introduction into basic DVC features. It doesn't
 go into details much, but provides links and expandable sections to learn more.
 
-At the very end there are a few complete examples to give you more hands-on
-experience with real life scenarios. The first one is about model and dataset
-[versioning](/doc/tutorials/versioning), and the second one is focused on
-[pipelines and reproducibility](/doc/tutorials/pipelines).
+You'll need [Git](https://git-scm.com) to run the commands in this tutorial.
```

Comment:
`in this tutorial` -> `in this section` or `in these chapters`

Reply:
don't like section or chapters - sounds too formal? Will try to come up with some alternative

Reply:
We call Get Started a "section" (of the docs) in a bunch of other places already. It's better than calling it a tutorial. Open to suggestions, lmk in order to review the term "section" throughout. Thanks

Reply:
How about guide?

I think the usage may vary depending on the context. And it's totally fine. The primary goal is the best experience for the reader. And, specifically, for the first time reader here. No point in limiting us to the rules for the sake of rules - it can create some bottlenecks down the road. 

So, in this specific case I would try to answer - what is easier to read and understand when you open the docs for the first time - section or tutorial or even guide? Or may be something else even?

What do you think?

Reply:
Def. not tutorial. I do like "guide". Would be good to review the existing instances of "section" though (migrate them to "guide" where appropriate).

I also don't dislike `run the commands in the following chapters`.

Reply:
Updated! My 2cs though - I think we indeed spend too much time debating this. I don't feel the difference or importance to be strict about the tutorial vs guide for the Get started. I'm pretty sure it's perceived as the same thing more or less.

In general I would say I care about consistency in the command reference most when we talk about DVC features - DVC-file, pipeline, etc. It's really not important when we talk about the documentation in the one of the supposed to reader friendly piece - get started index page.

Reply:
> I care about consistency in the command reference most when we talk about DVC features - DVC-file, pipeline, etc.

I like this approach, I can definitely be less strict, it will make maintaining the docs easier so I welcome this.

Still I think using terminology too loosely could result in confusing or unprofessionally looking text. Let's just use common sense? (Without trying to cross check every other doc for consistency) **As long as the terminology is correct then it should be fine.**

For example calling Get Started a "tutorial" is correct if you look at the definition of "tutorial". BUT I would still try to avoid it if possible to avoid confusion, since we have a separate section called Tutorials and Get Started isn't inside. If it's hard to avoid, then OK no problem.

Reply:
🤝